pkg/operator/sync_openshiftcontrollermanager_v311_00_test.go

```diff
@@ -483,12 +640,15 @@ func TestProgressingCondition(t *testing.T) {
 			}
 			controllerManagerOperatorClient := operatorfake.NewSimpleClientset(operatorConfig)
 
+			configInformers := configinformers.NewSharedInformerFactory(configClientFake, 10*time.Minute)
+			configLister := configInformers.Config().V1().ClusterVersions().Lister()
 			operator := OpenShiftControllerManagerOperator{
 				kubeClient:           kubeClient,
 				configMapsGetter:     kubeClient.CoreV1(),
 				proxyLister:          proxyLister,
 				recorder:             events.NewInMemoryRecorder(""),
 				operatorConfigClient: controllerManagerOperatorClient.OperatorV1(),
+				configLister:         configLister,
```

Comment:
You can build the lister directly as in https://github.com/openshift/library-go/blob/d7e7beca5baea998564ed16bd3d7e7dffbc7c01f/pkg/operator/configobserver/featuregates/simple_featuregate_reader_test.go#L270-L279. E.g.
```go
indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
indexer.Add(cv)
configlistersv1.NewClusterVersionLister(indexer)
```

Reply:
please check